api/v1/services/RegistrationService.js

```diff
@@ -321,9 +318,9 @@ module.exports.updateMentor = (mentor, attributes) => {
  * @throws {InvalidParameterError} when an attendee exists for the specified user
  */
 module.exports.createAttendee = (user, attributes) => {
-  if (!_hasValidAttendeeAssignment(attributes.projects, attributes.ecosystemInterests)) {
-    const message = 'One project or ecosystem interest must be provided';
-    const source = ['projects', 'ecosystemInterests'];
+  if (!_hasValidAttendeeAssignment()) {
+    const message = 'One interest must be provided';
+    const source = [];
```

Comment:
Are there any other related fields that might be necessary? OSContributor is one that comes to mind.

Reply:
Added OSContrutor in 6a2e2c84e513e9b0f05760bc0318d111b439fc34